osquery/CMakeLists.txt

```diff
@@ -223,8 +223,8 @@ if(NOT DEFINED ENV{SKIP_TESTS})
   # osquery core set of unit tests build with SDK.
   add_executable(osquery_tests main/tests.cpp ${OSQUERY_TESTS})
   TARGET_OSQUERY_LINK_WHOLE(osquery_tests libosquery)
-  target_link_libraries(osquery_tests gtest libosquery_testing)
-  SET_OSQUERY_COMPILE(osquery_tests "${CXX_COMPILE_FLAGS} -DGTEST_HAS_TR1_TUPLE=0")
```

Comment:
Was `-DGTEST_HAS_TR1_TUPLE=0` doing something important that I missed? When that flag is used, linking with gmock fails because apparently it requires the `TR1` tuple. On Mac, removing it solves the problem. On Ubuntu I get other errors, so still working on what the correct option is.